packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
Yes, you're right about that. Let me modify my objection: what happens if you run the following code:

```
var C = Foo.find({}, {limit: 10});
Meteor.defer(function () {
  console.log(C.fetch()); // will this fetch the cursor with the limit or not?
});
var count = C.count({applySkipLimit: false});
```